### PR TITLE
Fixes #21200 - gracefully handle long welcome text

### PR DESCRIPTION
**Pull request by tstrachota**
Fixes login page layout when the welcome text is too long.

**At wider screens**:
The welcome text is limited to 10 lines, scrollbar is displayed for longer texts.
before:
wide_before
after:
wide_after

**At narrow screens:**
The login page is extended according to the welcome message's length (the logo with helmet is scrolled up outside of the viewport at the screenshots).
before:
narrow_before
after:
narrow_after

### CODE REVIEW

**Comment by danseethaler** [comment]
I'll go ahead and review this PR

**Review by waldenraines — APPROVED** [review]
This works nicely for me and the changes look good.

**Comment by waldenraines** [comment]
> I'll go ahead and review this PR

Sorry, I didn't see this until I had already submitted my review

**Comment by danseethaler** [comment]
Yup - looks good to me too!

**Comment by ohadlevy** [comment]
[test foreman]

**Comment by ares** [comment]
[test foreman]

**Comment by tstrachota** [comment]
[test foreman]

**Comment by waldenraines** [comment]
@ares @timogoebel @ohadlevy anything else you'd like to see here before we merge?

**Comment by timogoebel** [comment]
Fine with me, but I don't know enough about our CSS to feel comfortable merging this.

**Comment by waldenraines** [comment]
> Fine with me, but I don't know enough about our CSS to feel comfortable merging this.

@timogoebel I know a little bit 😄  and it all looks fine to me.

**Comment by timogoebel** [comment]
> @timogoebel I know a little bit 😄 and it all looks fine to me.

I know. It's not, that I trust you guys with this (I do), but I still think you have the right to a review by a maintainer who knows that stuff better than I do.

**Pull request merged**
